src/common/src/config.rs

```diff
@@ -1515,6 +1520,14 @@ pub mod default {
         pub fn max_prefetch_block_number() -> usize {
             16
         }
+
+        pub fn fetch_unit() -> usize {
```

Comment:
64 may be too large.
As s3 is suggested to acquire object no small than 2MB, we can set it 32 by default.

Reply:
What about using 1 by default for now? It will fall back to the original behavior. We can quickly verify some use cases with 1.10 and tune it later. cc @hzxa21